src/cli/common/scripts-info.ts

```diff
@@ -99,3 +99,11 @@ const _scripts = {
 
 export const scripts = _scripts as Record<keyof typeof _scripts, ScriptInformation>
 
+export function getScriptInformation(scriptName: string): ScriptInformation | undefined{
+	for(const [script, info] of Object.entries(scripts)){
+		if(script === scriptName){
+			return info
+		}
+	}
+	return undefined
+}
```

Comment:
isn't this similar to just writing:

```suggestion
export function getScriptInformation(scriptName: string): ScriptInformation | undefined {
    return scripts[scriptName];
}
```

Reply:
You could, additionally use a `keyof` construct on `_scripts`/`scripts` to help the type system with the keys allowed.

Reply:
hmm the suggested change does not compile.
I am not entirely sure what you mean with "keyof construct"

Reply:
i mean typos can always happen but it should work just like that it is used elsewhere this way.